src/main/java/target/TargetRetryPolicy.java

```diff
@@ -30,6 +30,7 @@ public <T> RetryPolicy<T> get(ConsumerRecord<String, String> record, final ToInt
 
         return new RetryPolicy<T>()
             .withBackoff(delay, maxDelay, ChronoUnit.MILLIS, delayFactor)
+            .handleIf(e -> true)
```

Comment:
Why we need this?

Reply:
This is what tells [failesafe](https://failsafe.dev/policies/#failure-handling) to retry also exceptions